### ISSUE TITLE
Search broken

### ISSUE THREAD

**Issue by yonas** [axllent/mailpit#31]
```
INFO[2023/01/10 22:48:43] [smtp] starting on 0.0.0.0:1025              
INFO[2023/01/10 22:48:43] [http] starting server on http://0.0.0.0:8025/ 
ERRO[2023/01/10 22:52:10] sql: Scan error on column index 5, name "FromJSON": converting NULL to string is unsupported 
```

go version go1.19.4 freebsd/amd64
mailpit version: 47a556d05e68f4ce59548ea423b11dcad9ea2830

**Comment by axllent** [comment]
Thanks for the report.

This was briefly an issue _before_ v1.2.8 which should have been [fixed](https://github.com/axllent/mailpit/commit/ba24d145ffcbf8920889f27d85087b3498c0df4e) in early November last year. 

Could you please provide me some more information @yonas ...

Do you currently have old emails in your database what may have been capture before v1.2.8?

If some of your captured mail was from < v1.2.8 then there isn't anything I can do about it unfortunately as the data was stored as a `null` when there was no From address, rather than an empty JSON object (this breaks the SQL JSON search which is the error you are seeing) :-( If it was captured in v1.3.5 recently, can you please provide me with a copy of the full email (even if you change the values) so I can test please - assuming you can work out which one it is?

**Comment by yonas** [comment]
Hi @axllent

> Do you currently have old emails in your database what may have been capture before v1.2.8?

No, this is a fresh install.

**Comment by axllent** [comment]
@yonas Do you happen to know what email is causing this, and if so, are you able to provide me a copy of the raw email itself?

**Comment by axllent** [comment]
Thanks for the feedback! From what I can tell, all emails should contain a `From:` header - however the RFC documentation is very difficult to read and isn't crystal clear (to me) as to what exactly it should contain. Anyway, I was able to replicate the issue on my end following your instructions and have just pushed a fix for it to correctly index a missing From header. You'll find the fix in the latest release (v1.3.6).

**Comment by yonas** [comment]
Thanks @axllent , it works! :partying_face:
